src/nitpick/style/core.py

```diff
@@ -93,10 +93,12 @@ def find_initial_styles(self, configured_styles: Sequence[str], base: str | None
 
         """
         project_root = self.project.root
+        base_url = furl(base or project_root.resolve().as_uri())
 
         if configured_styles:
             chosen_styles = configured_styles
-            logger.info(f"Using styles configured in {PYPROJECT_TOML}: {chosen_styles}")
+            config_file = base_url.path.segments[-1] if base else PYPROJECT_TOML
```

Comment:
> * It always claimed the configured styles came from pyproject.toml even when
>   .nitpick.toml is actually used.

True, when I added `.nitpick.toml`  I forgot to change the logs.

It's been a while since I don't even look at logs, so thanks for noticing this!